docs/triptych-react-app-testing.md

```diff
@@ -0,0 +1,18 @@
+# Triptych React app testing
```

Comment:
Should we keep this committed in the final changeset? It seems more like a note specific to the process of handing off this PR.